students/wonjae/users/views.py

```diff
@@ -29,3 +29,20 @@ def post(self, request):
             
         except KeyError:
             return JsonResponse({"message" : "Key Error !!!"}, status=400)
+
+class LoginView(View):
+    def post(self, request):
+    
+        try:
+            data = json.loads(request.body)
+            
+            if not User.objects.filter(email=data['email']).exists():
+                return JsonResponse({"message" : "INVALID_USER"}, status=401)
+                
+            if not User.objects.get(email=data['email']).password == data['password']:
+                return JsonResponse({"message" : "INVALID_USER"}, status=401)
```

Comment:
line 39~40 코드는 line 42~43 코드와 같은 에러 값을 반환하기 때문에, email만 검증해줄 필요가 없을 것 같고,
아래와 같이 수정하면 좋을 것 같습니다!

```suggestion
            if not User.objects.filter(email=data['email'], password == data['password']).exists():
                return JsonResponse({"message" : "INVALID_USER"}, status=401)
```

Reply:
멘토님! 제가 원래 처음에 지금 코드 고쳐주신대로 해서 실행해봤었는데
그러면 제 이메일에 다른 사람 비밀번호를 넣어도 로그인이 되더라구요!
다시 한번만 확인 부탁드립니다! 감사합니다!

Reply:
실행결과와 코드 코멘트로 남겨주시겠어요?
데이터베이스 users 테이블도요!

Reply:
아 멘토님 제가 다른 코드와 착각했었습니다...
제가 원래 처음에 했던 코드는 아래와 같았습니다.
if not User.objects.filter(email=data['email']).exists() and not User.objects.filter(password=data['password']).exists()
멘토님께서 고쳐주신대로 다시 해서 실행결과 같이 첨부하겠습니다.

Reply:
<img width="1310" alt="DB users table" src="https://user-images.githubusercontent.com/56948394/178134329-6b56cddd-2fd5
<img width="1099" alt="case1" src="https://user
<img width="1101" alt="case2" src="https://user-images.githubusercontent.com/56948394/178134338-ff5c98a2-4b02-4a2a-b706-dfe1f55ecae4.png">
-images.githubusercontent.com/56948394/178134335-a9e27daf-f7d6-48fa-bd06-522071765074.png">
-4e33-9a0f-797f7aac2f43.png">

Reply:
![image](https://user-images.githubusercontent.com/56948394/178134413-8c38a9ac-3864-420e-b51c-87e7e0dc5c10.png)

Reply:
![image](https://user-images.githubusercontent.com/56948394/178134478-26fc8f1f-0b07-44d7-9f90-41a3261725a7.png)

Reply:
![image](https://user-images.githubusercontent.com/56948394/178134528-dff72817-ec00-4651-8989-23587790d62d.png)

Reply:
👍🏻